manifests/base/params.env

```diff
@@ -1,21 +1,24 @@
+odh-generic-data-science-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:029e48d075fefd4a9c399dd177fc805713b8d39434ae8142a1ee341ecd10c52d
 odh-tensorflow-gpu-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:c162a01a795ece51e57bfb7a926d05edf8bbe5189ee894a897dba60fc2b3d288
 odh-tensorflow-gpu-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:c162a01a795ece51e57bfb7a926d05edf8bbe5189ee894a897dba60fc2b3d288
 odh-trustyai-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:d0a41619ed23287b76e74ac859c2d8730903866a9e32eb1ac7d4138d2222f632
-odh-trustyai-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:20437e21d37d907e1287fd9a5c9d9ff7f52f3b14c8d74ac637c0fbe6dd242ce3
+odh-trustyai-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:d0a41619ed23287b76e74ac859c2d8730903866a9e32eb1ac7d4138d2222f632
 odh-codeserver-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:176bfa2e9c1f53d483d6ce4038897d19c34870b89e0c32ba87e94c00052acf2e
-odh-codeserver-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:6e78c8fb3fe1771d81d62377263e59007e5a3c2e7c24aac4498ef840d3aad369
+odh-codeserver-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:176bfa2e9c1f53d483d6ce4038897d19c34870b89e0c32ba87e94c00052acf2e
 odh-rstudio-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:562fde56f25507435520920ac41edd9ced0e025198a56142c44692f6340e0f0f
-odh-rstudio-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:0ea229bce375ca833a8036caa0d195ffc55e55843781e12b6694295194a6b7a3
+odh-rstudio-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:562fde56f25507435520920ac41edd9ced0e025198a56142c44692f6340e0f0f
 odh-rstudio-gpu-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:f2988efa77ba4725a701f5717e8b70b5141ad0cb04dfc42a68293752b28324a7
 odh-rstudio-gpu-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:f2988efa77ba4725a701f5717e8b70b5141ad0cb04dfc42a68293752b28324a7
 odh-rocm-minimal-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:8ab93581ef381c47a6bf2dc1162c7374f4e44966ce2fe383ed8ccf23c3ecfd32
+odh-rocm-minimal-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:8ab93581ef381c47a6bf2dc1162c7374f4e44966ce2fe383ed8ccf23c3ecfd32
 odh-rocm-pytorch-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:ad938f7194632a5e03d64412f537812b1b477a15c5866d3d8310c6ff27ff4ec2
+odh-rocm-pytorch-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:ad938f7194632a5e03d64412f537812b1b477a15c5866d3d8310c6ff27ff4ec2
+odh-rocm-tensorflow-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:55088024f2151777c5cd7ec0b5f7b80e3f724a150c43b1a268b4f64e2b36a72e
 odh-rocm-tensorflow-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:55088024f2151777c5cd7ec0b5f7b80e3f724a150c43b1a268b4f64e2b36a72e
```

Comment:
this doesn't seem right

Reply:
but the variable names are same, ther is no `n-1` on the last line, that's what I meant

Reply:
Ohhh right, thanks. fixed